server/middlewares/validateType.js

```diff
@@ -0,0 +1,13 @@
+const isValidType = (req, res, next) => {
+  const acceptedType = ['federal', 'legislature', 'state', 'local'];
+
+  if (!acceptedType.includes(req.body.type)) {
+    return res.status(400).send({
+      status: 400,
+      error: 'Type not of accepted value',
```

Comment:
Please specify the allowed types, so you don't leave the user confused.

Reply:
okay